src/HealthBase.jl

```diff
@@ -2,8 +2,19 @@ module HealthBase
 
 using Base: get_extension
 
+# GIVING AN PRECOMPILING ERROR, if we do Tables in here
+# Issue regarding the dependencies i believe
+
+# using DataFrames
```

Comment:
For this, I would not want us adding DataFrames.jl as a direct dependency so we can remove that. Having Tables as a dependency is fine as well as Base: @kwdef.